### PR TITLE
Add privacy policy

### LINKED ISSUE #523: Add updated privacy policy
- Use the programmatically generated data definitions so we don't get out of sync. 
- Create Page

Hold off on updated Content:
https://docs.google.com/document/d/1THDb1gqtY-fjC1HgDPOi1Y7pG-D8NgpRutrAheuDN2E/edit#

### PR DESCRIPTION
**Pull request by bamader**
Refactors the data definitions table to be a re-usable component, since we now want that same content on two separate pages. Adds an updated privacy policy with language drawn from the doc in the respective ticket. Closes #523

### CODE REVIEW

**Review by JordanGuinn — APPROVED** [review]
Looks good to me 👍🏽👍🏽👍🏽

File: client/src/containers/DataRequirements/DataRequirements.tsx [hunk 1]
```diff
@@ -84,35 +16,9 @@ const DataRequirements: React.FC = () => {
...
-              id="data-requirements-table"
-              data={sectionData}
-              rowKey={(row) => (row ? row.formattedName : '')}
-              columns={columns}
-              defaultSortColumn={0}
-            />
-          </div>
-        )
-      )}
+      <DataDefTable />
```
Comment (mmazanec22) [inline client/src/containers/DataRequirements/DataRequirements.tsx:19]: Small nit: abbreviations can be hard to follow when you're not the person who wrote them, and I feel like "definitions" is not an expected enough word to justify using an abbreviation

File: client/src/components/dataDefinitionsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import React, { useState, useContext, useEffect } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { ColumnMetadata } from '../shared/models';
+import AuthenticationContext from '../contexts/AuthenticationContext/AuthenticationContext';
+import { Table, Column } from '@ctoec/component-library';
+import { apiGet } from '../utils/api';
+
+const DataDefTable: React.FC = () => {
```
Comment (mmazanec22) [inline client/src/components/dataDefinitionsTable.tsx:8]: File name should match component name

File: client/src/components/dataDefinitionsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    return _bySection;
+  }, columnMetadataBySection);
+
+  return (
+    <div>
+      {Object.entries(columnMetadataBySection).map(
+        ([sectionName, sectionData]) => (
+          <div key={sectionName} className="margin-top-4">
+            <h2>{sectionName}</h2>
```
Comment (mmazanec22) [inline client/src/components/dataDefinitionsTable.tsx:79]: The privacy policy nests this component under an h2, so this header in that context needs to be an h3 instead.  IDK how this is nested in its other uses but probably it makes sense to have the header level be something a dev passes in.  There's an example of that in the component library.

File: client/src/components/dataDefinitionsTable.tsx (hunk 2 above)
Comment (jlhogan) [reply client/src/components/dataDefinitionsTable.tsx:8]: also, capitialize the file name for consistency with other components

**Review by jlhogan — APPROVED** [review]
